dataactcore/models/stagingModels.py

```diff
@@ -19,6 +19,8 @@ class FlexField(Base):
     row_number = Column(Integer, nullable=False)
     header = Column(Text)
     cell = Column(Text)
+    file_type_id = Column(Integer)
+    file_letter_name = Column(Text)
```

Comment:
I can look into the lookup thing, that would make sense. Adding a relationship seems like overkill since it would require a foreign key